File: app/src/main/java/br/com/jonathanzanella/myexpenses/chargeable/Chargeable.java
package br.com.jonathanzanella.myexpenses.chargeable;

public interface Chargeable {
	String getUuid();
	ChargeableType getChargeableType();
	String getName();
	boolean canBePaidNextMonth();
	void debit(int value);
	void credit(int value);
}

File: app/src/main/kotlin/br/com/jonathanzanella/myexpenses/chargeable/Chargeable.kt
package br.com.jonathanzanella.myexpenses.chargeable

interface Chargeable {
    val uuid: String?
    val chargeableType: ChargeableType
    var name: String?

    fun canBePaidNextMonth(): Boolean
    fun debit(value: Int)
    fun credit(value: Int)
}
